_unittest/test_12_PostProcessing.py

```diff
@@ -317,6 +317,59 @@ def test_09e_add_line_from_equation(self):
         assert new_report.create()
         assert new_report.add_limit_line_from_equation(1, 20, 0.5, "GHz")
 
+    @pytest.mark.skipif(
+        config["desktopVersion"] < "2022.2", reason="Not working in non-graphical mode in version earlier than 2022.2."
```

Comment:
Would the reason be better stated as "X does not work in non-graphical mode in a version earlier than 2022.2?
I'm not exactly sure what doesn't work--the report creation?